titus_isolate/metrics/constants.py

```diff
@@ -57,3 +57,8 @@
 
 RECONCILE_SKIP_COUNT = 'titus-isolate.reconcileSkipCount'
 RECONCILE_SUCCESS_COUNT = 'titus-isolate.reconcileSuccessCount'
+
+OVERSUBSCRIBE_FAIL_COUNT = 'titus-isolate.oversubscribeFailCount'
+OVERSUBSCRIBE_SKIP_COUNT = 'titus-isolate.oversubscribeSkipCount'
+OVERSUBSCRIBE_SUCCESS_COUNT = 'titus-isolate.oversubscribeSuccessCount'
+OVERSUBSCRIBE_RECLAIMED_CPU_COUNT = 'titus-isolate.oversubscribeReclaimedCpuCount'
```

Comment:
This is awesome.  If we update the `Workload` object as suggested elsewhere, we can also report `oversubscribeConsumedCpuCount` based on the sum of Workload opportunistic thread counts.

Reply:
added